templates/quintupleo.yaml

```diff
@@ -69,6 +69,14 @@ parameters:
     type: string
     default: CentOS-7-x86_64-GenericCloud-1503
 
+  user_data:
```

Comment:
Can we rename these to undercloud_user_data and undercloud_user_data_format so it's clear what they apply to?